commands/createfixpullrequests_test.go

```diff
@@ -116,20 +124,23 @@ func getMavenFixPackageVersionFunc() func(test packageFixTest) CreateFixPullRequ
 func TestFixPackageVersion(t *testing.T) {
 	currentDir, testdataDir := getTestDataDir(t)
 	for _, test := range packageFixTests {
-		// Create temp technology project
-		projectPath := filepath.Join(testdataDir, test.technology.ToString())
-		tmpProjectPath, cleanup := testdatautils.CreateTestProject(t, projectPath)
-		defer cleanup()
-		assert.NoError(t, os.Chdir(tmpProjectPath))
+		func() {
+			// Create temp technology project
+			projectPath := filepath.Join(testdataDir, test.technology.ToString())
+			tmpProjectPath, cleanup := testdatautils.CreateTestProject(t, projectPath)
+			test.testPath = tmpProjectPath
+			defer cleanup()
+			assert.NoError(t, os.Chdir(tmpProjectPath))
 
-		t.Run(test.technology.ToString(), func(t *testing.T) {
-			cfg := test.fixPackageVersionCmd(test)
-			// Fix impacted package for each technology
-			assert.NoError(t, cfg.updatePackageToFixedVersion(test.technology, test.impactedPackaged, test.fixVersion, test.packageDescriptor, tmpProjectPath))
-			file, err := os.ReadFile(test.packageDescriptor)
-			assert.NoError(t, err)
-			assert.Contains(t, string(file), test.fixVersion)
-		})
+			t.Run(test.technology.ToString(), func(t *testing.T) {
+				cfg := test.fixPackageVersionCmd(test)
+				// Fix impacted package for each technology
+				assert.NoError(t, cfg.updatePackageToFixedVersion(test.technology, test.impactedPackaged, test.fixVersion))
+				file, err := os.ReadFile(test.packageDescriptor)
+				assert.NoError(t, err)
+				assert.Contains(t, string(file), test.fixVersion)
+			})
+		}()
 	}
 	assert.NoError(t, os.Chdir(currentDir))
```

Comment:
This line should probably put inside a defer right after -
```
currentDir, testdataDir := getTestDataDir(t)
```

Reply:
done